doc/devdocs/readme.md

```diff
@@ -99,8 +113,8 @@ Our installer is two parts, an EXE and an MSI.  The EXE contains the MSI and han
 
 ### Compiling Bootstrapper Installer (.EXE)
```

Comment:
I'd change this to "Locally compiling bootstrapper installer

Reply:
OK, sounds good.